packages/react-router/src/ReactRouter/Router.tsx

```diff
@@ -97,10 +97,31 @@ class RouteManager extends React.Component<RouteComponentProps, RouteManagerStat
     const viewStackKeys = viewStacks.getKeys();
 
     viewStackKeys.forEach(key => {
-      const { view: enteringView, viewStack: enteringViewStack, match } = viewStacks.findViewInfoByLocation(location, key);
-      if (!enteringView || !enteringViewStack) {
+      const { view: _enteringView, viewStack: enteringViewStack, match } = viewStacks.findViewInfoByLocation(location, key);
```

Comment:
used variables should not be prefixed with _. Just find a better naming if you need the same name twice - or use let instead of const for enteringView

Reply:
Thanks,
I renamed the variable and I also added some comments.